gen3_tracker/common/__init__.py

```diff
@@ -319,9 +320,42 @@ def create_id(resource, project_id) -> str:
     from gen3_tracker import ACED_NAMESPACE
     assert resource, "resource required"
     assert project_id, "project_id required"
-    # identifier string is not unique when it complies with FHIR server. Using something else
-    # identifier_string = identifier_to_string(resource.identifier)
-    return str(uuid.uuid5(ACED_NAMESPACE, f"{project_id}/{resource.resource_type}/{resource}"))
+    identifier_string = identifier_to_string(resource.identifier)
+    return str(uuid.uuid5(ACED_NAMESPACE, f"{project_id}/{resource.resource_type}/{identifier_string}"))
+
+
+def create_object_id(path: str, project_id: str) -> str:
+    """Create a unique did for this object within a project."""
+    from gen3_tracker import ACED_NAMESPACE
```

Comment:
Just curious, is this import placed in the function so it's scoped to the method call?

Reply:
I was trying to improve cli startup time by `Lazy imports`
https://peps.python.org/pep-0690/#example

Reply:
nice, didn't know thanks!